apps/wing/src/commands/__snapshots__/test.test.ts.snap

```diff
@@ -0,0 +1,19 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`printing test reports > resource traces are not shown if debug mode is disabled 1`] = `
+"fail ┌ hello.w » root/env0/test:test
+     │ sleeping for 500 ms
```

Comment:
dem misaligned lines
![](https://www.reactiongifs.com/wp-content/uploads/2013/06/gag.gif)

We should eventually turn the md snapshot thing into something we can use outside of hangar